components/Pagination/PaginationActions.js

```diff
@@ -15,18 +20,22 @@ function TablePaginationActions({ count, page, rowsPerPage, onPageChange }) {
     }, [onPageChange, page])
 
     return (
-        <div style={{ display: "flex" }}>
+        <div style={{ display: "flex" }} className={classes.nextButton}>
```

Comment:
you can add the display property in the same class; also, the class should have a more relevant name as it's wrapping the pagination actions, not only the next button